documentation/CONFIG_OUTLIERS.md

```diff
@@ -24,6 +25,9 @@ The different types of detection models that can be configured are listed below.
 
 - **terms models**:  the terms model looks for outliers by calculating rare combinations of a certain field(s) in combination with other field(s). Example use case: tag all events that represent Windows network processes that are rarely observed across all reporting endpoints in order to detect C2 phone home activity.
 
+- **sudden_appearance models**: the sudden_appearance model looks for outliers by finding te sudden appearance of a 
+certain field(s).
+
```

Comment:
Like in the other sections, I propose you add an example here. Can you add the following:

Example use case: detect the sudden appearance of a new type of network traffic, or the sudden appearance of a Downloads directory from which processes are being executed.